# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../koffice/koffice-1.5.1-kexi-form-plugins.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur koffice-1.5.1-orig/kexi/plugins/forms/Makefile.am koffice-1.5.1/kexi/plugins/forms/Makefile.am
--- koffice-1.5.1-orig/kexi/plugins/forms/Makefile.am	2006-05-10 18:19:58.000000000 +0000
+++ koffice-1.5.1/kexi/plugins/forms/Makefile.am	2006-06-06 22:17:09.000000000 +0000
@@ -23,9 +23,10 @@
 	kexidataprovider.cpp kexiformscrollview.cpp kexiformeventhandler.cpp \
 	kexidbtextwidgetinterface.cpp kexiactionselectiondialog.cpp kexiformmanager.cpp \
 	kexidatasourcepage.cpp kexiformpart.cpp kexiformview.cpp
-libkexiformutils_la_LDFLAGS = $(all_libraries) $(VER_INFO) -Wnounresolved
+libkexiformutils_la_LDFLAGS = $(all_libraries) $(VER_INFO)
 libkexiformutils_la_LIBADD = $(top_builddir)/kexi/core/libkexicore.la \
 	$(top_builddir)/kexi/widget/libkexiextendedwidgets.la \
+	$(top_builddir)/kexi/formeditor/libkformdesigner.la \
 	$(top_builddir)/kexi/plugins/forms/widgets/libkexiformutilswidgets.la
 
 kformdesignerservicesdir=$(kde_servicesdir)/kformdesigner
diff -Nur koffice-1.5.1-orig/kexi/plugins/reports/Makefile.am koffice-1.5.1/kexi/plugins/reports/Makefile.am
--- koffice-1.5.1-orig/kexi/plugins/reports/Makefile.am	2006-05-10 18:20:02.000000000 +0000
+++ koffice-1.5.1/kexi/plugins/reports/Makefile.am	2006-06-06 22:17:09.000000000 +0000
@@ -5,24 +5,29 @@
 kexihandler_report_la_SOURCES = kexireports.cpp
 kexihandler_report_la_LDFLAGS = $(all_libraries) $(KDE_PLUGIN) $(VER_INFO) -module -no-undefined
 kexihandler_report_la_LIBADD = ../../core/libkexicore.la \
-  ../../widget/utils/libkexiguiutils.la \
-  ../../widget/libkexiextendedwidgets.la \
-  ../../formeditor/libkformdesigner.la \
-  ../forms/libkexiformutils.la \
+  $(top_builddir)/kexi/widget/utils/libkexiguiutils.la \
+  $(top_builddir)/kexi/widget/libkexiextendedwidgets.la \
+	$(top_builddir)/kexi/formeditor/libkformdesigner.la \
+  $(top_builddir)/kexi/plugins/forms/libkexiformutils.la \
   ./libkexireportutils.la
 
 kexireportwidgets_la_LDFLAGS =  $(all_libraries) $(KDE_PLUGIN) $(VER_INFO) -module
 kexireportwidgets_la_SOURCES =   reportwidgets.cpp kexireportfactory.cpp
 kexireportwidgets_la_LIBADD = $(top_builddir)/kexi/formeditor/libkformdesigner.la \
+	$(top_builddir)/kexi/plugins/forms/libkexiformutils.la \
+	$(top_builddir)/kexi/plugins/forms/widgets/libkexiformutilswidgets.la \
+	$(top_builddir)/kexi/widget/libkexiextendedwidgets.la \
 	./libkexireportutils.la
 
 lib_LTLIBRARIES = libkexireportutils.la
 libkexireportutils_la_SOURCES = \
 	kexireportpart.cpp kexireportview.cpp kexireportform.cpp
-libkexireportutils_la_LDFLAGS = $(all_libraries) $(VER_INFO) -Wnounresolved
+libkexireportutils_la_LDFLAGS = $(all_libraries) $(VER_INFO)
 libkexireportutils_la_LIBADD = $(top_builddir)/kexi/core/libkexicore.la \
+	$(top_builddir)/kexi/formeditor/libkformdesigner.la \
+	$(top_builddir)/kexi/plugins/forms/widgets/libkexiformutilswidgets.la \
 	$(top_builddir)/kexi/widget/libkexiextendedwidgets.la \
-	$(top_builddir)/kexi/plugins/forms/widgets/libkexiformutilswidgets.la
+	$(top_builddir)/kexi/plugins/forms/libkexiformutils.la
 
 kformdesignerservicesdir=$(kde_servicesdir)/kformdesigner
 kformdesignerservices_DATA = kformdesigner_kexireportfactory.desktop
